packages/gsm.rb

```diff
@@ -0,0 +1,242 @@
+          cp $? $@
+          chmod 444 $@
+       
+       $(GSM_INSTALL_MAN)/gsm.3:	$(MAN)/gsm.3
+      -		-rm $@
+      +		-rm $(RMFLAGS) $@
+          cp $? $@
+          chmod 444 $@
+       
+       $(GSM_INSTALL_MAN)/gsm_option.3:	$(MAN)/gsm_option.3
+      -		-rm $@
+      +		-rm $(RMFLAGS) $@
+          cp $? $@
+          chmod 444 $@
+       
+       $(GSM_INSTALL_MAN)/gsm_explode.3:	$(MAN)/gsm_explode.3
+      -		-rm $@
+      +		-rm $(RMFLAGS) $@
+          cp $? $@
+          chmod 444 $@
+       
+       $(GSM_INSTALL_MAN)/gsm_print.3:	$(MAN)/gsm_print.3
+      -		-rm $@
+      +		-rm $(RMFLAGS) $@
+          cp $? $@
+          chmod 444 $@
+       
+       $(GSM_INSTALL_INC)/gsm.h:	$(INC)/gsm.h
+      -		-rm $@
+      -		cp $? $@
+      -		chmod 444 $@
+      -
+      -$(GSM_INSTALL_LIB)/libgsm.a:	$(LIBGSM)
+      -		-rm $@
+      +		-rm $(RMFLAGS) $@
+          cp $? $@
+          chmod 444 $@
+       
+      +$(GSM_INSTALL_LIB)/libgsm.so:	$(LIBGSMSO)
+      +		-rm $(RMFLAGS) $@ $@.1 $@.1.0.13
+      +		cp $?.1.0.13 $@.1.0.13
+      +		chmod 755 $@.1.0.13
+      +		$(LN) libgsm.so.1.0.13 $@
+      +		$(LN) libgsm.so.1.0.13 $@.1
+       
+       # Distribution
+       
+      @@ -425,7 +425,7 @@
+            -print | xargs rm $(RMFLAGS)
+       
+       clean:	semi-clean
+      -		-rm $(RMFLAGS) $(LIBGSM) $(ADDTST)/add		\
+      +		-rm $(RMFLAGS) $(LIBGSMSO)* $(ADDTST)/add		\
+            $(TOAST) $(TCAT) $(UNTOAST)	\
+            $(ROOT)/gsm-1.0.tar.Z
+       
+      @@ -473,22 +473,22 @@
+       $(TST)/test-result:	$(TST)/lin2cod $(TST)/cod2lin $(TOAST) $(TST)/run
+            ( cd $(TST); ./run ) 
+       
+      -$(TST)/lin2txt:		$(TST)/lin2txt.o $(LIBGSM)
+      +$(TST)/lin2txt:		$(TST)/lin2txt.o $(LIBGSMSO)
+            $(LD) $(LFLAGS) -o $(TST)/lin2txt \
+      -				$(TST)/lin2txt.o $(LIBGSM) $(LDLIB)
+      +				$(TST)/lin2txt.o $(LIBGSMSO) $(LDLIB)
+       
+      -$(TST)/lin2cod:		$(TST)/lin2cod.o $(LIBGSM)
+      +$(TST)/lin2cod:		$(TST)/lin2cod.o $(LIBGSMSO)
+            $(LD) $(LFLAGS) -o $(TST)/lin2cod \
+      -				$(TST)/lin2cod.o $(LIBGSM) $(LDLIB)
+      +				$(TST)/lin2cod.o $(LIBGSMSO) $(LDLIB)
+       
+      -$(TST)/gsm2cod:		$(TST)/gsm2cod.o $(LIBGSM)
+      +$(TST)/gsm2cod:		$(TST)/gsm2cod.o $(LIBGSMSO)
+            $(LD) $(LFLAGS) -o $(TST)/gsm2cod \
+      -				$(TST)/gsm2cod.o $(LIBGSM) $(LDLIB)
+      +				$(TST)/gsm2cod.o $(LIBGSMSO) $(LDLIB)
+       
+      -$(TST)/cod2txt:		$(TST)/cod2txt.o $(LIBGSM)
+      +$(TST)/cod2txt:		$(TST)/cod2txt.o $(LIBGSMSO)
+            $(LD) $(LFLAGS) -o $(TST)/cod2txt \
+      -				$(TST)/cod2txt.o $(LIBGSM) $(LDLIB)
+      +				$(TST)/cod2txt.o $(LIBGSMSO) $(LDLIB)
+       
+      -$(TST)/cod2lin:		$(TST)/cod2lin.o $(LIBGSM)
+      +$(TST)/cod2lin:		$(TST)/cod2lin.o $(LIBGSMSO)
+            $(LD) $(LFLAGS) -o $(TST)/cod2lin \
+      -				$(TST)/cod2lin.o $(LIBGSM) $(LDLIB)
+      +				$(TST)/cod2lin.o $(LIBGSMSO) $(LDLIB)
+    GSM_PATCH_EOF
```

Comment:
Why not just download the patch from the source?  I'm guessing you needed some changes to the patch?

Reply:
Wanted to avoid another download, but that seems silly.